lib/bemxjst/index.js

```diff
@@ -426,13 +426,12 @@ BEMXJST.prototype.tryRun = function(context, ent) {
   try {
     return ent.run(context);
   } catch (e) {
-    console.error('BEMXJST ERROR: cannot render ' +
-      [
-        'block ' + context.block,
-        'elem ' + context.elem,
-        'mods ' + JSON.stringify(context.mods),
-        'elemMods ' + JSON.stringify(context.elemMods)
-      ].join(', '), e);
+    context.onError({
+      block: context.block,
```

Comment:
I still think `context` is better than `{block, elem, mods, elemMods}` object.

Reply:
@sbmaxx what's your opinion?

Reply:
good idea!